mynearwallet/my-near-wallet
# feat: add bridged and native token functionality

## Description
https://meteorlabs.jetbrains.space/p/main/issues/159

## Work Done
* Bundler auto installed string_decoder due to a dependency conflict
* Added language state to storybook
* Added env variable to jest
* Converted components to tsx where necessary
* Added stories 
* Merged `BalanceDisplay` into`Balance` component since it was pointless to have both
* Removed `SendConfirmModal` component since it is not used
* Added token utils function that applied bridge/native naming to tokens 
* Implemented the utility wherever there was an API call to retrieve tokens
* Changed EnterAmount component in Send to show notification when Bridged token is used

## How To Test
* Run the app 
* Look at list of tokens wherever you can find them
* Go to send 
* In storybook you can see the notification in `EnterAmount` under `withWarning` (Note there is an issue with translations due to the old translations package)

## Screenshot
<img width="609" alt="image" src="https://user-images.githubusercontent.com/46312751/234285303-5947dd5f-7f31-42b4-860c-339d7f80091e.png">

<img width="569" alt="image" src="https://user-images.githubusercontent.com/46312751/234285425-299f2d2f-1e3d-4258-bbf3-3596686a2d4e.png">

<img width="568" alt="image" src="https://user-images.githubusercontent.com/46312751/234285634-96443fb4-0773-419f-ad6a-42508d36e756.png">

<img width="585" alt="image" src="https://user-images.githubusercontent.com/46312751/234285546-8f95013b-ec9d-4786-a25a-881f3c8505b3.png">
